packages/openneuro-app/src/scripts/dataset/components/ValidationBlock.tsx

```diff
@@ -38,6 +40,40 @@ export const ValidationBlock: React.FC<ValidationBlockProps> = ({
   issuesStatus,
   validation,
 }) => {
+  const { stopPolling, startPolling } = useContext(DatasetQueryContext)
+
+  // Function to stop polling if issuesStatus or validation exists
+  const stopPollingValidation = () => {
+    if (issuesStatus || validation) {
+      if (typeof stopPolling === "function") {
+        try {
+          stopPolling()
+        } catch (error) {
+          Sentry.captureException(error)
+        }
+      }
+    }
+  }
+
+  // Function to start polling if ValidationPending is shown.
+  const startPollingValidation = () => {
+    if (!issuesStatus && !validation) {
+      if (typeof startPolling === "function") {
+        try {
+          startPolling(500) // 500ms poll interval
```

Comment:
```suggestion
          startPolling(10000) // 10s poll interval
```

This looks good but I'd raise the polling time to avoid flooding the server with API requests if validation is taking a long time.